content/faq/voice/text-to-speech-android-tts/index.fr.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Synthèse vocale sur Android
+description: Guide pour utiliser la synthèse vocale sur Android
+aliases: ["/fr/faq/synthèse-vocale-ttsw-sur-android"]
+
+taxonomies:
+  faq: ["Instructions vocales"]
```

Comment:
Note that the same taxonomy should be used in another faq article.

Reply:
I based it on this :

https://github.com/organicmaps/organicmaps.github.io/blob/master/content/faq/voice/cant-hear-instructions/index.fr.md?plain=1#L7